pkg/meta/tkv.go

```diff
@@ -184,6 +184,7 @@ All keys:
   SHssssssss         session heartbeat // for legacy client
   SIssssssss         session info
   SSssssssssiiiiiiii sustained inode
+  Niiiiiiii	         detached inodes
```

Comment:
align

Reply:
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/31313340/228137212-93717117-c763-4cb5-876d-7a784348bdda.png">
<img width="1544" alt="image" src="https://user-images.githubusercontent.com/31313340/228137227-f3521777-bfb1-4943-ac32-728d0862b15b.png">

github display cannot be aligned with the local IDE at the same time

Reply:
vim
<img width="1325" alt="image" src="https://user-images.githubusercontent.com/31313340/228138077-014ba981-05b0-4c75-ba70-ff4bbc5a905f.png">
I think the local ide presentation is right.

Reply:
Maybe it was a special character problem before, so delete it and rewrite it